assets/src/domain/eventEditor/ui/datetimes/datesList/List.tsx

```diff
@@ -1,43 +1,46 @@
-import React, { CSSProperties } from 'react';
+import React from 'react';
 import { __ } from '@wordpress/i18n';
-import { H3 } from '@blueprintjs/core/lib/esm';
+import { Divider, Typography } from 'antd';
 
-import DatesListFilterBar from './filterBar/DatesListFilterBar';
 import AddNewDateButton from './AddNewDateButton';
-import DatetimeCard from './cardView/DateCard';
-import { Datetime } from '@edtrServices/apollo/types';
-
-const listStyle: CSSProperties = {
-	display: 'flex',
-	flexFlow: 'row wrap',
-	justifyContent: 'space-between',
-	width: '100%',
-};
+import DatesListEntityFilters from './filterBar/DatesListEntityFilters';
+import { Datetime } from '../../../services/apollo/types';
+import { EntityList } from '@appLayout/entityList';
+
+import { DateCardList } from './cardView';
+import { TableView } from './tableView';
+
+import useDatesListFilterState from './filterBar/useDatesListFilterState';
 
 interface ListProps {
 	datetimes: Datetime[];
 }
 
-const List: React.FC<ListProps> = ({ datetimes }) => {
-	const header = <H3 style={{ margin: '2rem 0 .5rem' }}>{__('Dates List')}</H3>;
+const { Title } = Typography;
 
-	const datetimesList = (
-		<>
-			<div style={listStyle}>
-				{datetimes.map((date) => (
-					<DatetimeCard id={date.id} key={date.id} />
-				))}
-			</div>
-			<AddNewDateButton />
-		</>
+const List: React.FC<ListProps> = ({ datetimes }) => {
+	const { processedDates, ...entityFiltersProps } = useDatesListFilterState(datetimes);
+	const entityFilters = <DatesListEntityFilters {...entityFiltersProps} />;
+	const header = (
+		<Title level={3} style={{ marginBottom: '1.5rem' }}>
+			{__('Dates List')}
+		</Title>
```

Comment:
now that we are out of the prototype phase can we use css/sass and avoid inline styles plz?

as well, I'm ok with this JSX just being added directly to the render output below since it is pretty simple

Reply:
I wasn't sure at first what's the best approach to add space here:
- Had used `Divider` but it had added too much space
IMHO it would handy to have our own wrapper like `HeaderTitle` or smth that would apply that spacing for all sections headers, rather than manually passing that.

Reply:
For the time being, had pulled that out in a separate file.

Reply:
https://github.com/eventespresso/event-espresso-core/pull/2431/commits/f0f34cb0160e780724a73f38be3c58234d668019